src/smallestPrimeFactor.js

```diff
@@ -12,11 +12,42 @@
  * @returns {number} The smallest prime factor of num
  */
 function smallestPrimeFactor(num) {
+  let prime_factors = findPrimeFactors(num);
+
+  return prime_factors[0];
```

Comment:
This code tells me you understand that the _first_ factor you find will be prime. Why bother finding _every_ prime factor? Return the first factor larger than `1` that you find. If the input is prime, you'll wind up returning the input itself. Otherwise, you'll return the smallest factor (which _has_ to be prime).